eval/src/vespa/eval/instruction/generic_cell_cast.cpp

```diff
@@ -43,9 +43,11 @@ struct SelectGenericCellCastOp {
 InterpretedFunction::Instruction
 GenericCellCast::make_instruction(const ValueType &result_type,
                                   const ValueType &input_type,
+                                  CellType to_cell_type,
```

Comment:
we should be able to trust ValueType::cell_cast and not have to pass the result_type